### ISSUE TITLE
Classification on a new Dataset

### ISSUE THREAD

**Issue by YuvalRom** [jcyang34/lspin#1]
I read the article and the results are amazing!
I would like to try it on a different problem with a different dataset.
Could you explain what changes I will need to make for one of the demos in order to do so?

**Comment by jcyang34** [comment]
Thanks for your interest!

I think the major change is to modify the data matrices and labels to yours in the DataSet_meta object:

```
dataset = DataSet_meta(**{'_data':X_train, '_labels':y_train,'_meta':y_train,
                '_valid_data':X_valid, '_valid_labels':y_valid,'_valid_meta':y_valid,
                '_test_data':X_test, '_test_labels':y_test,'_test_meta':y_test})
```

where _\*_data are the input data matrices (with shape n \* m), _\*_labels are the corresponding labels (with shape n*1 if it's binary).

As for _\*_meta, these are the additional metadata labels as a measure of the affinities between the samples, in case one wants to enable the second penalty term to force similar samples to select similar features. To enable it, one needs to set ```compute_sim = True``` in ```model.train()``` and specify the regularization parameter ```gamma1``` in ```model```. Here in this implementation, the affinity will be 1 if two samples are of the same labels, and 0 otherwise.
